src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt

```diff
@@ -118,8 +124,11 @@ fun createLookupElement(element: RsCompositeElement, scopeName: String): LookupE
 fun LookupElementBuilder.withPriority(priority: Double): LookupElement =
     PrioritizedLookupElement.withPriority(this, priority)
 
-private val InsertionContext.isInUseBlock: Boolean
-    get() = file.findElementAt(startOffset - 1)!!.parentOfType<RsUseItem>() != null
+private val InsertionContext.getUseItem: RsUseItem
+    get() = PsiTreeUtil.findElementOfClassAtOffset(file, tailOffset - 1, RsUseItem::class.java, false)!!
```

Comment:
Hm, this might thrown a null pointer exception at runtime. Kotlin's `!!` are very similar to Rust's `.unwraps()`: can help to get the job done, but might cause your program to die in flames in production. 

The safer way would be:

```kotlin
                val useItem = context.getUseItem()
                if (useItem != null) {
                    // Kotlin nows that useItem is not null here, so it does not require `!!` after `useItem`.
                    val hasSemicolon = useItem.lastChild!!.elementType == RsElementTypes.SEMICOLON
                    if (!(hasSemicolon || context.isInGlob)) {
                        context.addSuffix(";")
                    }
                    return@handler
}

...

private fun InsertionContext.getUseItem(): RsUseItem
    = PsiTreeUtil.findElementOfClassAtOffset(file, tailOffset - 1, RsUseItem::class.java, false)
```